bundle/manifests/flows.netobserv.io_flowcollectors.yaml

```diff
@@ -3014,6 +3014,11 @@ spec:
                         - ENABLED
                         type: string
                     type: object
+                  enable:
+                    default: true
+                    description: enable the console plugin deployment. spec.Loki.enable
+                      must be true
```

Comment:
Is it better if it says must *also* be true?

Reply:
Why not :smile: 
https://github.com/netobserv/network-observability-operator/pull/354/commits/d902fc6e33ac430faffbcc9b59e9cd49f9745b1f